geomesa-lambda/geomesa-lambda-datastore/src/main/scala/org/locationtech/geomesa/lambda/data/LambdaDataStore.scala

```diff
@@ -35,29 +34,22 @@ import org.opengis.filter.Filter
 
 import scala.concurrent.duration.Duration
 
-class LambdaDataStore(
-    val persistence: DataStore,
-    producerConfig: Map[String, String],
-    consumerConfig: Map[String, String],
-    offsetManager: OffsetManager,
-    config: LambdaConfig)
-   (implicit clock: Clock = Clock.systemUTC()
-   ) extends DataStore with HasGeoMesaStats with LazyLogging {
+class LambdaDataStore(val persistence: DataStore, config: LambdaConfig)(implicit clock: Clock = Clock.systemUTC())
+    extends DataStore with HasGeoMesaStats with LazyLogging {
 
   import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
 
   import scala.collection.JavaConverters._
 
   private val authProvider: Option[AuthorizationsProvider] = persistence match {
-    case ds: AccumuloDataStore => Some(ds.config.authProvider)
+    // this is a bit of a hack to work around hbase null vs empty auths
+    case ds: GeoMesaDataStore[_] if ds.config.authProvider.getAuthorizations != null => Some(ds.config.authProvider)
```

Comment:
Ok, so ds.config.authProvider is never null here?

Reply:
this is scala! things are never null! :D
the other option would be to have some kind of 'empty' auth provider and match on the specific class.